src/assets/styles/index.css

```diff
@@ -1,6 +1,7 @@
+/* purgecss start ignore */
 @tailwind base;
-
 @tailwind components;
+/* purgecss start ignore */
```

Comment:
Shouldn't this be `/* purgecss end ignore */` ?

Reply:
My bad, thanks, fixed.